boards/nordic/bm_nrf54l15dk/bm_nrf54l15dk_nrf54l15_cpuapp_s115_softdevice_mcuboot.dts

```diff
@@ -70,9 +70,9 @@
 			};
 		};
 
-		slot0_partition: partition@b000 {
+		slot0_partition: partition@9c00 {
```

Comment:
I'm not able to boot the application with this partition address. If I change the address to 0xa000 it starts fine.

I think the problem is that 0x9c00 is not 2KiB aligned. So assuming an image header of 2KiB, the interrupt table at the start of application is not properly aligned.

Same for L05 and L10.